Test/ProteinParsimonyTest.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using EngineLayer;
+using NUnit.Framework;
+using Proteomics.ProteolyticDigestion;
+using TaskLayer;
+using IO.MzML;
+using MassSpectrometry;
+using Org.BouncyCastle.Asn1.X509;
+using System.Text.RegularExpressions;
+
+namespace Test
+{
+    [TestFixture]
+    public class ProteinParsimonyTest
```

Comment:
Tests take 25 seconds to run due to IO code that is duplicated and re-run for every test. Need to break out code shared between runs into [OneTimeSetup ](https://docs.nunit.org/articles/nunit/writing-tests/attributes/onetimesetup.html)and [OneTimeTearDown](https://docs.nunit.org/articles/nunit/writing-tests/attributes/onetimeteardown.html) methods to reduce the time it takes to run these tests.